en-US/D.xml

```diff
@@ -189,7 +189,13 @@
 			<term>dialog box</term>
 			 <listitem>
 				<para>
-					Refer to the Word Usage chapter of the <citetitle>IBM Style Guide</citetitle> for usage information related to this and similar terms.
+					<emphasis>n.</emphasis> Use "dialog box" (<emphasis>not</emphasis> "dialog", "dialogue", "dialogue box", or "window"), to refer to an element that is displayed for a user to interact with a graphical user interface.
```

Comment:
General comment: I wonder whether we should include a statement similar to [Microsoft's "dialog" entry](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dialog-box-dialog-dialogue), which emphasizes the learner's actions over specifying the UI element. Something like: "In general, avoid naming specific UI elements. Instead, identify the user's actions. If you must refer to a dialog, then use..."

Reply:
tbh I'm surprised to see this, because "referring to IBM" suggests using "dialog" and not "dialog box", which we don't do.

In some cases, yes, there's no need to name GUI objects, such as Go to File > Save. In other cases, not so much. I'm thinking of all the times we refer to icons, links, tabs, panes, and whatever inside UIs, and omitting object element names could get confusing really quickly. Even in the case of menus, we often write "On the whatever menu, click foo and then click bar to open the Pandora box." (Maybe that could be improved, sure; I'm just not comfortable with the idea of dropping object names to save a few wordswhen I see so much opportunity for lost clarity.)

Reply:
We need to update this entry (if not remove it altogether), because IBM dropped "dialog box" in favour of "dialog" a while ago, so it no longer works to point to IBM for guidance here.

Reply:
Right. They changed; we didn't. So yes we need to remove the reference to their guidance because it no longer applies to us. We've always used "dialog box" and as I mentioned even elected not to change when the topic came up before. So yes, remove the reference to IBM and just say that "dialog box" is correct. Maybe provide a couple of examples of what not to use, which many ppl find helpful.